src/agentscope/memory/memory.py

```diff
@@ -63,7 +68,7 @@ def delete(self, index: Union[Iterable, int]) -> None:
     @abstractmethod
     def load(
         self,
-        memories: Union[str, dict, list],
+        memories: Union[str, list, MessageBase],
         overwrite: bool = False,
```

Comment:
Add more docstring to describe the arguments. For example, if `overwrite` is `False` and the content of memory is not empty, how will we load the new memory (to the beginning or end)?

Reply:
Similar problem with the export interface, e.g. what's the use of the argument `to_mem`?

A suggestion: place `file_path` before `to_mem`, which is a more common used argument.